cli/file_watcher.rs

```diff
@@ -344,8 +343,8 @@ where
 fn new_watcher(
   sender: Arc<mpsc::UnboundedSender<Vec<PathBuf>>>,
 ) -> Result<RecommendedWatcher, AnyError> {
-  let mut watcher: RecommendedWatcher =
-    Watcher::new(move |res: Result<NotifyEvent, NotifyError>| {
+  let watcher = Watcher::new(
```

Comment:
I'm a bit weary about this change... I don't remember exact reason we didn't want to update watcher crate now, so let's see if any problem surface